python/python/bystro/ancestry/train.py

```diff
@@ -308,6 +313,88 @@ def _calc_fst(variant_counts: pd.Series, samples: pd.DataFrame) -> float:
     return (p * (1 - p) - total) / (p * (1 - p))
 
 
+def load_1kgp_vcf(vcf_filepath: str) -> pd.DataFrame:
+    """Temporary placeholder method to load in 1kgp vcf
+    filtered down to the same variants as the gnomad loadings for testing
+    using plink2.
+    """
+    #This line will have to be altered if testing other vcfs
+    header_vcf=pd.read_csv(vcf_filepath,delimiter='\t',skiprows=107)
+    dosage_vcf = header_vcf.replace("0|0", 0)
+    dosage_vcf = dosage_vcf.replace("0|1", 1)
+    dosage_vcf = dosage_vcf.replace("1|0", 1)
+    dosage_vcf = dosage_vcf.replace("1|1", 2)
+    #If testing other vcfs this line will also need to be altered
+    dosage_vcf=dosage_vcf.rename(
+        columns={"#CHROM": "Chromosome", "POS": "Position"}, 
+        errors="raise"
+    )
+    return dosage_vcf
```

Comment:
What would you think about setting dosage_vcf so that its index is sample_ids and its columns are variants?  That could simplify the overlap calculation (see below)